### PR TITLE
TECH-529 microsite page template footer fix, cms reorganizing and fav…

### PR DESCRIPTION
**Pull request by cjmabry**
…icon support

### CODE REVIEW

File: pages/models.py [hunk 1]
```diff
@@ -199,41 +206,65 @@ class MicrositePage(Page):
...
-            FieldPanel('facebook_url'),
-            FieldPanel('standard_footer_show'),
-            FieldPanel('custom_footer_show'),
-            FieldPanel('custom_footer_content'),
-            FieldPanel('custom_footer_background_color'),
-            FieldPanel('custom_footer_text_color'),
-            ImageChooserPanel('custom_footer_background_image'),
-        ]
+        ImageChooserPanel('custom_favicon'),
+        MultiFieldPanel(
```
Comment (ericbroder) [inline pages/models.py]: nice 😄

File: pages/templates/pages/base_page.html [hunk 2]
```diff
@@ -54,7 +54,9 @@
     <script src="{% static "dist/js/bundle.min.js" %}?v=12"></script>
     <link href="https://fonts.googleapis.com/css?family=Lato:400,700" rel="stylesheet">
     <meta name="google-site-verification" content="devyaHsbJD6w8nNEWsAu3SsODhEfijQqCOhFbuXdpCc" />
-    <link rel="icon" type="image/x-icon" href="{% static "dist/img/favicon.ico" %}" />
+    {% block favicon %}
+      <link rel="icon" type="image/x-icon" href="{% static "dist/img/favicon.ico" %}" />
```
Comment (ericbroder) [inline pages/templates/pages/base_page.html]: Maybe better to just put image url in block? Is anything else here ever going to change?

Comment (ericbroder) [reply pages/templates/pages/base_page.html]: Similar to the page title logic.

File: pages/templates/pages/microsite_page.html [hunk 3]
```diff
@@ -118,6 +130,15 @@
   </style>
 {% endblock %}
 
+{% block favicon %}
+  {% if page.custom_favicon %}
+    {% image page.custom_favicon original as favicon %}
+    <link rel="icon" type="image/x-icon" href="{{favicon.url}}" />
+  {% else %}
+    <link rel="icon" type="image/x-icon" href="{% static "dist/img/favicon.ico" %}" />
```
Comment (ericbroder) [inline pages/templates/pages/microsite_page.html]: Here also, just set variable for image url? Don't need to repeat same markup?

File: pages/templates/pages/microsite_page.html [hunk 4]
```diff
@@ -206,5 +227,7 @@
     <div class="microsite">
       {{ block.super }}
     </div>
+  {% else %}
+    {% include "partials/footer-js.html" %}
```
Comment (ericbroder) [inline pages/templates/pages/microsite_page.html]: Could you please add code comments to explain what is going on here? This is confusing.

Comment (ericbroder) [reply pages/templates/pages/microsite_page.html]: Is this a solution to the earlier discussion about decoupling footer scripts from footer content? Why can't we do it in more straightforward way?

**Comment by cjmabry** [comment]
Documenting these PRs so we can come back and fix the small things, thanks for review!

**Pull request merged**